vite.config.ts

```diff
@@ -2,7 +2,6 @@ import eslintPlugin from "@nabla/vite-plugin-eslint";
 import { TanStackRouterVite } from "@tanstack/router-plugin/vite";
 import react from "@vitejs/plugin-react";
 import { defineConfig } from "vite";
-import { nodePolyfills } from "vite-plugin-node-polyfills";
```

Comment:
Why was this removed? We need this to run knex (our SQL builder) in the browser.
I should add a comment about that since it's definitely not obvious why this is important